gen/components/NewContactForm.js

```diff
@@ -26,7 +26,7 @@ class NewContactForm extends React.Component {
         <TextInput
           onChangeText={(text) => this.setState({text})}
           ref={this.props.generateTestHook('NewContactForm.TextInput')}
-          value={this.state.text}
+          placeholder={this.state.text}
```

Comment:
Placeholder is the text that's displayed in the TextInput field - it's the same as doing {value='text'} but you don't have to backspace it in order to type stuff.